platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -130,6 +130,7 @@ COPY ["files/configdb-load.sh", "/usr/bin/"]
 COPY ["files/arp_update", "/usr/bin/"]
 COPY ["files/buffers_config.j2", "files/qos_config.j2", "/usr/share/sonic/templates/"]
 COPY ["files/sonic_version.yml", "/etc/sonic/"]
+COPY ["default_config_db.json", "/etc/sonic/"]
```

Comment:
Did we agree to change this file name to something more obvious for user?

Reply:
what about 'port_breakout_config_db.json'. I will change in Dev Code as well.

Reply:
+1 to "port_breakout_config_db.json"

Reply:
good, go ahead with this new name.